manifests/integreatly-mobile-developer-console/0.3.0/mobile-developer-console-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,270 @@
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - apps.openshift.io
+          resources:
+          - deploymentconfigs
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resourceNames:
+          - mobile-developer-console-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - patch
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - mdc.aerogear.org
+          resources:
+          - mobileclients
+          verbs:
+          - get
+          - list
+          - watch
+          - update
+          - patch
+        - apiGroups:
+          - push.aerogear.org
+          resources:
+          - pushapplications
+          - androidvariants
+          - iosvariants
+          - webpushvariants
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - mobile-security-service.aerogear.org
+          resources:
+          - mobilesecurityserviceapps
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          - prometheusrules
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - integreatly.org
+          resources:
+          - grafanadashboards
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        serviceAccountName: mobile-developer-console-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - mobile
+  maintainers:
+  - email: aerogear@googlegroups.com
+    name: AeroGear
+  maturity: alpha
+  provider:
+    name: AeroGear
+  replaces: mobile-developer-console-operator.v0.2.6
```

Comment:
If we're only including the latest csv version, the `replaces` tag should be removed, otherwise I believe OLM would require the previous csv package version also before installing/upgrading